.eslintrc.js

```diff
@@ -0,0 +1,121 @@
+const ERROR = "error";
+const WARN = "warn";
+const OFF = "off";
+
+const errorIfStrict = process.env.STRICT ? ERROR : WARN;
+
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+    // TODO EVG-17441: add Jest here.
+  },
+  parser: "@typescript-eslint/parser",
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    ecmaVersion: "latest",
+    sourceType: "module",
+  },
+  extends: [
+    "eslint:recommended",
+    "plugin:import/recommended",
+    "plugin:react/recommended",
+    "airbnb", // Airbnb includes some helpful rules for ESLint and React that aren't covered by recommended.
```

Comment:
I'm down to give Airbnb rules a try in Parsley -- if we don't like them we can disable them, and if we do we can add them to Spruce!